config/schedule.rb

```diff
@@ -154,6 +154,12 @@
     trigger: health_trigger,
     interruptable: true,
   },
+  # FIXME uncomment after launch
+  # {
+  #   task: 'driver:hmis_external_apis:ac_hmis:update_unit_availability',
+  #   frequency: 3.minutes,
+  #   trigger: ENV['HMIS_OKTA_CLIENT_ID'] == 'true' && ENV['RAILS_ENV'] == 'production'
+  # },
```

Comment:
We need this to be happening on day 1 of launch, it shouldn't require a new deploy. If there is no way to turn it on and off without doing a code deploy, we'll just have to leave it on.